cpp/oneapi/dal/algo/decision_forest/common.hpp

```diff
@@ -503,6 +514,17 @@ class descriptor : public detail::descriptor_base<Task> {
         return *this;
     }
 
+    /// Splitter strategy, if 'best', best treshold for each will
+    /// be selected. If 'random', trashold will be selected randomly.
+    splitter_mode get_splitter_mode() const {
+        return base_t::get_splitter_mode();
+    }
+
+    auto& set_splitter_mode(splitter_mode value) {
```

Comment:
Please add a comment describing the method.

Reply:
It is actually added with 3 lines above. It is aligned with rest methods: getters and setters are followed by single description.
Something like:
```cpp
// Description
Type get_...() {}

auto& set... () {}
```